core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedService.java

```diff
@@ -149,11 +141,16 @@
                       method.getDeclaringClass().getSimpleName(), method.getName());
         isKotlinSuspendingMethod = KotlinUtil.isSuspendingFunction(method);
         this.resolvers = requireNonNull(resolvers, "resolvers");
-        exceptionHandler =
-                new CompositeExceptionHandlerFunction(object.getClass().getSimpleName(), method.getName(),
-                                                      requireNonNull(exceptionHandlers, "exceptionHandlers"));
+
+        requireNonNull(exceptionHandlers, "exceptionHandlers");
+        if (exceptionHandlers.isEmpty()) {
+            exceptionHandler = NoopExceptionHandlerFunction.INSTANCE;
```

Comment:
Can't we just assign `null` instead of defining `NoopExceptionHandlerFunction` and using it?

Reply:
+1 That is also a good idea.